src/cineca/cineca.py

```diff
@@ -151,7 +151,7 @@ def main():
     writer.writerow(['IRI', 'Label', 'Definition', 'Parent', 'Synonym', 'Comment', 'Question Description',
                      'Expected Answer Type', 'See Also ID', 'Known Number of Cohorts', 'Use Cases Requirements'])
     writer.writerow(['ID', 'LABEL', 'A definition', 'SC %', 'A alternative term', 'A comment', 'A question description',
-                     'A answer type', 'A see also', 'A number of cohorts', 'A use cases requirements'])
+                     'A answer type', 'A see also SPLIT=/', 'A number of cohorts', 'A use cases requirements'])
```

Comment:
We usually split on `|`. Are they using `|` already?

Reply:
I just chose `/` because that's what they're using for multiple values in the Google sheet. I can replace uses of `/` with `|` if you prefer.

Reply:
Ok, that's fine.